.github/workflows/run_tests.yaml

```diff
@@ -1,30 +0,0 @@
-name: Run-Tests
```

Comment:
Hmm. I sort of liked the name run_tests.yaml. It is comprehensive enough to include both unit and integration tests.